acapy_plugin_toolbox/holder/v0_1/__init__.py

```diff
@@ -148,15 +147,40 @@ def send_condition(principal: Principal):
             await send_to_admins(session, message, responder, condition=send_condition)
 
 
+def send_condition(principal: Principal, record: PresExRecord, approved: set):
+    if "all-credentials" in principal.privileges:
+        return True
+    elif "limited-credentials" in principal.privileges:
+        cred_def_id_restrictions = set(record.credential_definition_id)
+        if cred_def_id_restrictions.issubset(approved):
+            return True
+        else:
+            return False
```

Comment:
This can be simplified to:
```suggestion
        return cred_def_id_restrictions.issubset(approved)
```